test/e2e/apimachinery/garbage_collector.go

```diff
@@ -457,8 +469,10 @@ var _ = SIGDescribe("Garbage collector", func() {
 		rcClient := clientSet.CoreV1().ReplicationControllers(f.Namespace.Name)
 		podClient := clientSet.CoreV1().Pods(f.Namespace.Name)
 		rcName := "simpletest.rc"
-		// TODO: find better way to keep this label unique in the test
-		uniqLabels := map[string]string{"gctest": "orphan_pods_nil_option"}
+		suffix := randomSuffix()
```

Comment:
how about extracting this as a method and turn those 4 into one line?

Reply:
Sure,good idea,Thanks